pkg/daemon/daemon.go

```diff
@@ -51,6 +53,8 @@ type Daemon struct {
 	client snclientset.Interface
 	// kubeClient allows interaction with Kubernetes, including the node we are running on.
 	kubeClient *kubernetes.Clientset
+	// kubeClient allows interaction with Kubernetes crd,s
+	dynamicclient dynamic.Interface
```

Comment:
There has already been an informer who monitors the SriovOperatorConfig CR. I don't think we need an extra client for the same task.